### ISSUE TITLE
about tf.reduce_max function

### ISSUE THREAD

**Issue by freefuiiismyname** [baidu/DuReader#14]
DuReader/tensorflow/layers/match_layer.py  line 94:
            b = tf.nn.softmax(tf.expand_dims(tf.reduce_max(sim_matrix, 2), 1), -1)
i think tf.reduce_max(t,2) hasnt been defined.it should be 0 or 1.

**Comment by freefuiiismyname** [comment]
sry.:(((
this is a stupid question cuz sim_matrix is 3d.
